src/main/java/io/github/tryexceptelse/jdex/JDex.java

```diff
@@ -92,7 +107,8 @@ private Dex buildDex() {
    * @return Ui: User Interface object
    */
   private Ui buildUi(){
-    return new Gui();
+    return new Gui(this); //placeholder.
```

Comment:
'this' is calling the Gui constructor with a reference to 'this' object, so that Gui can have a reference to manipulate the program